psi4/src/psi4/dfmp2/mp2.cc

```diff
@@ -187,7 +187,16 @@ void DFMP2::common_init()
 }
 double DFMP2::compute_energy()
 {
+
     print_header();
+    if (Ca_subset("AO","ACTIVE_VIR")->colspi()[0] == 0) {
+        energies_["Opposite-Spin Energy"] = 0.;
+        energies_["Same-Spin Energy"] = 0.;
+        energies_["Singles Energy"] = 0.;
+        outfile->Printf("No virtual orbitals.\n\n");
+        print_energies();
+        return energies_["Total Energy"];
+    }
```

Comment:
I still think this should fail as it has serious implications for anything that uses a `dfmp2` Wavefunction down the line. Gradients, natural orbitals, etc. We should catch the odd ball case here instead of having potential seg faults later in the code.